src/fmgc/src/components/FmsMessages.ts

```diff
@@ -155,6 +169,7 @@ enum FMMessageUpdate {
  */
 abstract class FMMessageSelector {
     abstract message: FMMessage;
+    abstract ndSide?: 'L' | 'R';
```

Comment:
**thought:** maybe rename this to `efisSide` since it can also be transmitted to the PFD

Reply:
I think we really need to treat ND, PFD, and MCDU messages separately in general. Because there are some messages that go to the MCDU, some that go to the ND, some that go to the PFD, and some that go to combination of those. That said, this is probably one that can be shared.